README.markdown

```diff
@@ -88,8 +88,8 @@ TABLE: weak_guy;
 +---------------+-------------+------+-----+---------+----------------+
 | Field         | Type        | Null | Key | Default | Extra          |
 +---------------+-------------+------+-----+---------+----------------+
-| strong_guy_id | int(10)     | NO   | PRI | 0       |                |
-| weak_guy_id   | int(10)     | NO   | PRI | NULL    | auto_increment |
+| strong_guy_id | int(10)     | NO   |     | 0       |                |
```

Comment:
This is incorrect. strong_guy_id is a primary key of the weak_guy table (it has a composite primary key because "weak_guy" is in a 1-to-many dependent relationship with "strong_guy" (hence the names strong and weak).